SwiftUIDZ/SwiftUIDZ/FiltersView.swift

```diff
@@ -0,0 +1,163 @@
+                    } label: {
+                        Image(systemName: Constants.backButtonIconName)
+                    }
+                    Spacer()
+                    Text(Constants.navigationTitle)
+                        .foregroundColor(.white)
+                        .font(.title3)
+                        .fontWeight(.bold)
+                    Spacer()
+                    Image(Constants.filtersImageName)
+                        .resizable()
+                        .renderingMode(.template)
+                        .foregroundColor(.white)
+                        .frame(width: 20, height: 20)
+                }
+            }
+        }
+        .accentColor(.white)
+    }
+
+    // MARK: - Private Properties
+
+    @Environment(\.presentationMode) var presentationMode
+
+    @StateObject private var viewModel = FiltersViewModel()
+
+    private var oneColumnGrid = [GridItem(.fixed(80))]
+    private var fiveColumnGrid = [
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70)),
+        GridItem(.fixed(70))
+    ]
+
+    private var colorsLazyVGrid: some View {
+        ScrollView(.vertical, showsIndicators: false) {
+            LazyVGrid(columns: fiveColumnGrid, spacing: 20) {
+                ForEach(Colors.colors, id: \.self) { color in
+                    Button {
+                        viewModel.color = color
+                    } label: {
+                        Circle()
+                            .fill(color)
+                            .frame(width: 40, height: 40)
+                    }
+                }
+            }
+            .padding()
+        }
+    }
+
+    private var colorsTextView: some View {
+        HStack {
+            Text(Constants.colorsText)
+                .foregroundColor(viewModel.color)
+                .fontWeight(.semibold)
+                .font(.title)
+                .padding()
+            Spacer()
+        }
+    }
+
+    private var sliderView: some View {
+        Slider(value: $viewModel.sliderValue, in: 100 ... 5000, step: 500)
+            .padding()
+    }
+
+    private var productLazyHGrid: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            LazyHGrid(rows: oneColumnGrid, spacing: 15) {
+                ForEach(Product.getProducts(), id: \.name) { product in
+                    ZStack {
+                        RoundedRectangle(cornerRadius: 20)
+                            .fill(.white)
+                            .shadow(color: Color(UIColor.systemGray4), radius: 6, x: 0, y: 2)
+                        Image(product.imageName)
+                            .resizable()
+                            .renderingMode(.template)
+                            .frame(width: 70, height: 70)
+                            .foregroundColor(Color(Constants.orangeColorName))
+                            .padding()
```

Comment:
```suggestion
                        RoundedRectangle(cornerRadius: 20)
                            .fill(.white)
                            .shadow(color: Color(UIColor.systemGray4), radius: 6, x: 0, y: 2)
                        Image(product.imageName)
                            .resizable()
                            .renderingMode(.template)
                            .frame(width: 70, height: 70)
                            .foregroundColor(Color(Constants.orangeColorName))
                            .padding()
```
я пишу отдельно, если не хочешь вынести, просто закрой, я больше таких комментов писать Не буду